config/v1/types_oauth.go

```diff
@@ -379,14 +341,14 @@ type RequestHeaderIdentityProvider struct {
 	//   https://www.example.com/sso-login?then=${url}
 	// ${query} is replaced with the current query string
 	//   https://www.example.com/auth-proxy/oauth/authorize?${query}
-	// Required when UseAsChallenger is set to true.
+	// Required when challenge is set to true.
 	ChallengeURL string `json:"challengeURL"`
 
-	// clientCA is a reference to a configmap with the trusted signer certs. If empty, no request
-	// verification is done, and any direct request to the OAuth server can impersonate any identity
-	// from this provider, merely by setting a request header.
+	// ca is a reference to a config map by name with the trusted signer certs.
```

Comment:
More CA words as above.. Something like:
"clientCA is a reference to a config map by name with a PEM-encoded CA. The CA is used by the identity provider as a trust anchor to perform verification on incoming requests, in order to prevent header spoofing."